src/Security/CMSSecurity.php

```diff
@@ -41,12 +44,27 @@ protected function init()
         Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/client/dist/js/vendor.js');
     }
 
+    public function login($request, $service = Authenticator::CMS_LOGIN)
+    {
+        return parent::login($request, Authenticator::CMS_LOGIN);
+    }
+
     public function Link($action = null)
     {
         /** @skipUpgrade */
         return Controller::join_links(Director::baseURL(), "CMSSecurity", $action);
     }
 
+    protected function getAuthenticator($name = 'cms')
```

Comment:
Parent method signature declares $name as non-optional. I suggest to change parent to Security::getAuthenticator($name = 'default')